autoload/neoformat.vim

```diff
@@ -227,8 +227,15 @@ function! s:generate_cmd(definition, filetype) abort
         return {}
     endif
     if !executable(executable)
-        call neoformat#utils#log('executable: ' . executable . ' is not an executable')
-        return {}
+        if &shell =~ '\v%(powershell|pwsh)'
```

Comment:
does this need to be here?

Reply:
Well to check for Powershell scripts, I would need to be in Powershell of course. Lemme reword it